docs/src/full tutorials/MNIST.md

```diff
@@ -0,0 +1,96 @@
+## Image Classification Example
+An expanded version of this example, with early stopping and
+snapshots, is available [here](/examples/mnist).
+
+We define a builder that builds a chain with six alternating
+convolution and max-pool layers, and a final dense layer, which we
+apply to the MNIST image dataset.
+
+First we define a generic builder (working for any image size, color
+or gray):
+
+```julia
+using MLJ
+using Flux
+using MLDatasets
+
+# helper function
+function flatten(x::AbstractArray)
+	return reshape(x, :, size(x)[end])
+end
+
```

Comment:
I'm working on an independent PR to update MNIST and I noticed the following changes are needed for latest versions of packages, to avoid some warnings. 

So:

- Get rid `flatten` definition. Instead `import MLUtils` and use `MLUtils.flatten` instead.
- Change way data is loaded as noted in suggestion below.

Will copy you into the PR when it's ready.

Reply:
I have hidden this tutorial anyway. Next PR will be focused on workflow examples and full tutorials